apps/authoring/src/renderer/pages/editor/elements/pane-details/elements/glossary/glossary-form.tsx

```diff
@@ -23,6 +23,14 @@ export const GlossaryForm = ({ term, onHide, onSubmit }: GlossaryFormProps) => {
   const handleFormSubmit = () => {
     onSubmit(data);
   };
+  const handleKeyboardSubmit = (e: React.KeyboardEvent) => {
+    const cancelButton = document.querySelector('.glossary-cancel-button');
+    const cancelIsFocused = document.activeElement === cancelButton;
+    if (e.key === 'Enter' && !cancelIsFocused) {
+      e.preventDefault();
+      onSubmit(data);
+    }
+  };
```

Comment:
I wasn't able to get an onSubmit method to work on the form, so I used an onKeyDown and wrote a custom key handler to submit the form when the user hits Enter (and the cancel button is not currently focused). This may be an issue for accessibility--we can optionally add another condition whereby Enter must be focused, and the user would need to tab down to it for Enter to submit the form. But really, we should be using an onSubmit if we can get it to work